### PR TITLE
Scalar type representation

### PR DESCRIPTION
**Pull request by codedmart**
### What
Return proper representation for scalar types. Still need to verify and/or fix serialization of types.

### CODE REVIEW

File: crates/configuration/src/introspection.rs [hunk 1]
```diff
@@ -64,3 +65,30 @@ pub struct IntrospectionForeignKeyColumn {
 pub struct IntrospectionSchema {
     pub name: String,
 }
+
+pub fn map_type_representation(sql_type: &str) -> Option<TypeRepresentation> {
+    match sql_type.to_lowercase().as_str() {
+        "bigint" => Some(TypeRepresentation::Int64),
```
Comment (danieljharvey) [inline crates/configuration/src/introspection.rs:71]: These all seem reasonably sensible to me.

File: crates/query-engine/sql/src/sql/helpers.rs [hunk 2]
```diff
@@ -50,12 +50,28 @@ pub fn make_column(
...
         alias,
-        Expression::ColumnReference(ColumnReference::TableColumn { table, name }),
+        cast_column(
+            column_type,
+            Expression::ColumnReference(ColumnReference::TableColumn { table, name }),
+        ),
     )
 }
+
+/// Should we cast the column to a specific type?
```
Comment (danieljharvey) [inline crates/query-engine/sql/src/sql/helpers.rs:64]:
```suggestion
/// Some types need to be stringified to match representations in `ndc-spec`, this function wraps those types in cast so they becomes strings in JSON.
```

File: crates/ndc-sqlserver/tests/snapshots/mutation_tests__stored_procedures__execute_stored_procedure_with_relationships.snap [hunk 3]
```diff
@@ -13,31 +13,31 @@ expression: result
             "rows": [
               {
                 "InvoiceId": 98,
-                "Total": 3.98
+                "Total": "3.98"
```
[the remark this answers is not in the document]
Comment (codedmart) [reply crates/ndc-sqlserver/tests/snapshots/mutation_tests__stored_procedures__execute_stored_procedure_with_relationships.snap:16]: 1.1?

[the remark this answers is not in the document]
Comment (codedmart) [reply crates/ndc-sqlserver/tests/snapshots/mutation_tests__stored_procedures__execute_stored_procedure_with_relationships.snap:16]: Kind of a bummer to go to 2.0 here now :(.

[the remark this answers is not in the document]
Comment (danieljharvey) [reply crates/ndc-sqlserver/tests/snapshots/mutation_tests__stored_procedures__execute_stored_procedure_with_relationships.snap:16]: Well it's a reasonably substantial runtime behaviour change, we need to clearly communicate it to users one way or another.

**Pull request merged**